src/Annotation/ApiResource.php

```diff
@@ -117,6 +121,34 @@ final class ApiResource
      */
     private $accessControlMessage;
```

Comment:
`$accessControl` and `$accessControlMessage` should be deleted, so the autocompletion will not be triggered anymore in PHPStorm.

Reply:
Shouldn't I tag them as deprecated instead?